Kasun1Don/pump.task
# Pump 66 project page nav

# Purpose

Updating the route to open the project task page when a new project is created.

[PUMP-66](https://labrys-intern.atlassian.net/browse/PUMP-66?atlOrigin=eyJpIjoiOTY0ODExYjhkODI2NDllOWI3NDUzMTNkODlhOGRhNWUiLCJwIjoiaiJ9)

# Approach

Use Path parameter routing rather than Query parameter routing.

# Testing


1. Navigate to the Projects page.
2. Create a new project and verify that the navigation to the tasks page works correctly.
3. Ensure that existing projects can be accessed and their IDs are correctly used for navigation.
